### ISSUE TITLE
Sticky menus

### ISSUE THREAD

**Issue by azmeuk** [pallets/pallets-sphinx-themes#33]
I could not find any previous discussion about this subject.
I would suggest making the side-menus sticky in most themes.

The side menu usually holds the TOC, and when reading something at the bottom of a very long page ([example](https://werkzeug.palletsprojects.com/en/1.0.x/routing/)), we need to scroll all the way up to find a link in the TOC. The default RTD theme has sticky menu, and I find this very convenient.

What do you think?

**Comment by davidism** [comment]
I've rejected this in the past because it adds a lot of complexity to the layout code, and when I've tried to implement it it's never behaved particularly well. See pallets/flask-sphinx-themes#10 and pallets/flask-sphinx-themes#11 for a previous attempt. Press the "Home" key, or "Shift + Space", to quickly return to the top of the page.

I am not really a CSS designer, and knowing all the corner cases that can happen, I wouldn't really trust / be able to review another person's implementation to be "correct" unless I already knew they were a CSS expert.
